### PR TITLE
Invites: Display different notices depending on the user role

### LINKED ISSUE #2176: People: Message after accepting invite always says 'user' not 'administrator', 'editor' etc
1. Accept invite as administrator of site
2. Message says "You're now a user of: e2eflowtesting"

Enhancement: It would be nice to state:

"You're now an administrator of: e2eflowtesting" or "You're now an editor of: e2eflowtesting" etc.

confirm message

### PR DESCRIPTION
**Pull request by lezama**
__How to test__

* checkout
* invite users with different user roles
* copy the invite email link and change wpcalypso.wordpress.com with calypso.dev:3000
* accept the invites
* __assert__ that different roles have different "invitation accepted" notices

Closes #2176

cc:
@rickybanister for improving wording
@ebinnion and @roccotripaldi for code review and testing :)

### CODE REVIEW

File: client/lib/invites/notices.js [hunk 1]
```diff
@@ -7,51 +7,131 @@ import get from 'lodash/object/get'
...
+					<p className="invite-message__intro">
+						{
+							i18n.translate(
+								'Since you\'re new, you might like to {{docsLink}}take a tour{{/docsLink}}.',
+								{ components: { docsLink: <a href="http://en.support.wordpress.com/" target="_blank" /> } }
+							)
+						}
+					</p>
+				</div>,
+				{ displayOnNextPage	}
```
Comment (ebinnion) [inline client/lib/invites/notices.js]: There's a tab after `displayOnNextPage` instead of a space.

File: client/lib/invites/notices.js [hunk 2]
```diff
@@ -7,51 +7,131 @@ import get from 'lodash/object/get'
...
+		case 'editor':
+			notices.success(
+				<div>
+					<h3 className="invite-message__title">
+						{ i18n.translate( 'You\'re now an Editor of: %(site)s', {
+							args: { site: get( invite, 'site.title' ) }
+						} ) }
+					</h3>
+					<p className="invite-message__intro">
+						{ i18n.translate( 'This is your site dashboard where you can write posts and control your site. ' ) }
```
Comment (ebinnion) [inline client/lib/invites/notices.js]: I'm not sure that an editor can "control a site". The [Codex](https://codex.wordpress.org/Roles_and_Capabilities) describes an editor as:

> somebody who can publish and manage posts including the posts of other users.

Here's an example of what we use in [invite-form-header](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/invites/invite-form-header/index.jsx#L207):

> As an editor, you will be able to publish and manage your own posts and the posts of others, as well as upload media.

File: client/lib/invites/notices.js [hunk 3]
```diff
@@ -7,51 +7,131 @@ import get from 'lodash/object/get'
...
+		case 'author':
+			notices.success(
+				<div>
+					<h3 className="invite-message__title">
+						{ i18n.translate( 'You\'re now an Author of: %(site)s', {
+							args: { site: get( invite, 'site.title' ) }
+						} ) }
+					</h3>
+					<p className="invite-message__intro">
+						{ i18n.translate( 'This is your site dashboard where you can write posts. ' ) }
```
Comment (ebinnion) [inline client/lib/invites/notices.js]: For what it's worth, an `author` can also publish posts.

**Comment by ebinnion** [comment]
I left some comments on wording. Besides that, this LGTM.

**Comment by ebinnion** [comment]
I updated some of the wording, but I'm not sure I like it to be honest. @rickybanister - would you mind reading over some of the wording updates in d23463c418825ecfc66ce72b0ee3d804940895fa please?

**Comment by lezama** [comment]
Thanks Eric!

**Comment by ebinnion** [comment]
Let's :ship: this. We can easily change the words later.

**Comment by alisterscott** [comment]
Tested this on localhost and wp-calypso and I can no longer see any message display upon accepting an invite (tested Editor and Author) and creating an account.

**Comment by ebinnion** [comment]
This is a known issue that @lezama is working on in #2273.

**Pull request merged**